sitewit/models.py

```diff
@@ -52,3 +54,62 @@ def create(cls, user, site, url, user_token=None):
             user.location, user_token=user_token)
 
         return Account(result['accountInfo'], user_data=result['userInfo'])
+
+    @classmethod
+    def get(cls, account_token):
+        """ Get SiteWit account by account token.
+
+        Args:
+            account_token (str): account token.
+
+        Returns:
+            Instance of Account class.
+
+        Raises:
+            demands.HTTPServiceError: if any error happened on HTTP level.
+        """
+        result = cls.get_service().get_account(account_token)
+
+        return Account(result)
+
+    @classmethod
+    def update(cls, account_token, url, country_code, currency):
+        """ Update SiteWit account with given data.
+
+        Args:
+            account_token (str): account token.
+            url (str): new URL.
+            country_code (str): new country code
+                (https://sandboxpapi.sitewit.com/Help/ResourceModel?
+                 modelName=CountryCode)
+            currency (str): new currency
+                (https://sandboxpapi.sitewit.com/Help/ResourceModel?
+                 modelName=BudgetCurrency
+
+        Returns:
+            Instance of account class.
+
+        Raises:
+            demands.HTTPServiceError: if any error happened on HTTP level.
+        """
+        result = cls.get_service().update_account(
+            account_token, url, country_code, currency)
+
+        return Account(result)
+
+    @classmethod
+    def delete(cls, account_token):
+        """ Get SiteWit account by account token.
+
+        Args:
+            account_token (str): account token.
+
+        Returns:
+            Instance of Account class.
+
+        Raises:
+            demands.HTTPServiceError: if any error happened on HTTP level.
+        """
+        result = cls.get_service().delete_account(account_token)
```

Comment:
API does this. Also, we had a discussion with @RayeN (Voffka) before and decided that alsays returning modified instance is a way-to-go.